data/katha.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 name: Katha Infocom Pvt. Ltd.
-description: Katha has multiple products that cater to media businesses and is run by people divided into teams of Content, Tech and Growth. On our mission to distribute unbiased and non-sensational information, we have currently partnered with more than 25 informational enterprises and cater to more than a million
+description: Katha has multiple products that cater to media businesses. On our mission to distribute unbiased and non-sensational information, we have currently partnered with more than 25 informational enterprises and cater to more than a million
 end users every months. Katha App is first audible news app in Malayalam having 100K+ downloads in Playstore.
```

Comment:
Could you remove the newline here. It's the small issue that is causing the error but the rest of data is perfect❤️